algorithmic_efficiency/workloads/criteo1tb/criteo1tb_pytorch/models.py

```diff
@@ -25,6 +40,127 @@ def forward(self, dense_features, sparse_features):
     return torch.cat((dense_features, interactions_flat), dim=1)
 
 
+class DLRMResNet(nn.Module):
+  """Define a DLRM-Small model.
+
+  Parameters:
+    vocab_size: vocab size of embedding table.
+    num_dense_features: number of dense features as the bottom mlp input.
+    mlp_bottom_dims: dimensions of dense layers of the bottom mlp.
+    mlp_top_dims: dimensions of dense layers of the top mlp.
+    embed_dim: embedding dimension.
+  """
+
+  def __init__(self,
+               vocab_size,
+               num_dense_features=13,
+               num_sparse_features=26,
+               mlp_bottom_dims=(256, 256, 256),
+               mlp_top_dims=(256, 256, 256, 256, 1),
+               embed_dim=128,
+               dropout_rate=0.0,
+               use_layer_norm=False,
+               embedding_init_multiplier=None):
+    super().__init__()
+    self.vocab_size = torch.tensor(vocab_size, dtype=torch.int32)
+    self.num_dense_features = num_dense_features
+    self.num_sparse_features = num_sparse_features
+    self.mlp_bottom_dims = mlp_bottom_dims
+    self.mlp_top_dims = mlp_top_dims
+    self.embed_dim = embed_dim
+
+    # Ideally, we should use the pooled embedding implementation from
+    # `TorchRec`. However, in order to have identical implementation
+    # with that of Jax, we define a single embedding matrix.
+    num_chucks = 4
```

Comment:
is this supposed to be "num_chunks" (here and below)?

Reply:
Yup it sure is lol. Typo from other PR.